ui/src/views/Denylist/ViewDenylistAddress/index.jsx

```diff
@@ -0,0 +1,109 @@
+import { hot } from 'react-hot-loader';
+import React, { Component, Fragment } from 'react';
+import { graphql, withApollo } from 'react-apollo';
+import Spinner from '@mozilla-frontend-infra/components/Spinner';
+import Dashboard from '../../../components/Dashboard';
+import DenylistForm from '../../../components/DenylistForm';
+import ErrorPanel from '../../../components/ErrorPanel';
+import formatError from '../../../utils/formatError';
+import denylistAddressQuery from './denylistAddress.graphql';
+import addAddressQuery from './addAddress.graphql';
+import deleteAddressQuery from './deleteAddress.graphql';
+
+@hot(module)
+@withApollo
+@graphql(denylistAddressQuery, {
+  skip: ({ match: { params } }) => !params.notificationAddress,
```

Comment:
This view will only render when there is a notification address so this check is probably not necessary. I think it's safe to remove the `skip` field.

Reply:
I think we use the same view for adding new addresses in which case we need to skip the query